src/main/java/org/broadinstitute/hellbender/tools/spark/sv/FindBadGenomicKmersSpark.java

```diff
@@ -31,7 +32,9 @@
     private static final long serialVersionUID = 1L;
     @VisibleForTesting static final Long MAX_KMER_FREQ = 3L;
     private static final int REF_RECORD_LEN = 10000;
-    private static final int CHUNK_SIZE = 3000000;
+    // to give a task size of 100Kb -- the max size recommended in the Spark documentation
+    private static final int REF_RECORDS_PER_PARTITION = 102400 / REF_RECORD_LEN;
```

Comment:
Does this stop that warning?  I understood that as the task itself was too large, (i.e. the code being serialized to run the task), rather than the partition size.  It may be that something is unexpectedly being serialized that shouldn't be if you're seeing that?